build/Targets/Settings.targets

```diff
@@ -165,7 +165,7 @@
   <!-- Import the props files from the toolset NuGet packages if we're not in a bootstrap build -->
   <ImportGroup Label="GlobalNuGets" Condition="'$(BootstrapBuildPath)' == ''">
     <Import Project="$(ToolsetCompilerPropsFilePath)"
-            Condition="Exists('$(ToolsetCompilerPropsFilePath)') And '$(OS)' == 'Windows_NT'" />
+            Condition="'$(BuildingProject)' == 'true' And Exists('$(ToolsetCompilerPropsFilePath)') And '$(OS)' == 'Windows_NT'" />
```

Comment:
This would mean you won't have your build task being loaded now during the design time build, correct?